src/agent/sqlite_manager/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.24)
+
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
Many options are already in `src/cmake/CommonSettings.cmake`. Check for example: https://github.com/wazuh/wazuh-agent/blob/master/src/agent/agent_info/CMakeLists.txt#L8-L9

Reply:
Done